kwc-blockly-flyout.js

```diff
@@ -288,6 +288,7 @@ Polymer({
       this._removeBlockListeners();
 
       oldBlocks.forEach(block => {
+          if (!block.rendered) return;
```

Comment:
```
if (!block.rendered) {
    return;
}
```

Reply:
?

Reply:
sure, I prefer the top pattern if it's just for a return but happy to do the more verbose if we'd prefer